apps/messagegui/app.js

```diff
@@ -267,11 +269,27 @@ function showMessageSettings(msg) {
     /*LANG*/"View Message" : () => {
       showMessageScroller(msg);
     },
+    /*LANG*/"Reply": () => {
```

Comment:
Please could you add a comment here like `// deleted below if reply not supported` as it might be a bit confusing for others (it was for me!).

Reply:
I've changed this to use an if statement instead just so it's more in line with the surrounding logic